MC/config/PWGHF/pythia8/generator/pythia8_charm_with_mudecays_Mode2.cfg

```diff
@@ -0,0 +1,72 @@
+### authors: Fabrizio Grosa (fabrizio.grosa@cern.ch)
+###          Cristina Terrevoli (cristina.terrevoli@cern.ch)
+###          Fabio Catalano (fabio.catalano@cern.ch)
+###          Nicole Bastid (nicole.bastid@cern.ch)
+### charm decay muons 
+### last update:  November 2024
+
+### beams
+Beams:idA 2212			# proton
+Beams:idB 2212 			# proton
+Beams:eCM 13600. 		# GeV
+
+### processes
+SoftQCD:inelastic on		# all inelastic processes
+
+### decays
+ParticleDecays:limitTau0 on	
+ParticleDecays:tau0Max 10.
+
+### switching on Pythia Mode2
+ColourReconnection:mode 1
+ColourReconnection:allowDoubleJunRem off
+ColourReconnection:m0 0.3
+ColourReconnection:allowJunctions on
+ColourReconnection:junctionCorrection 1.20
+ColourReconnection:timeDilationMode 2
+ColourReconnection:timeDilationPar 0.18
+StringPT:sigma 0.335
+StringZ:aLund 0.36
+StringZ:bLund 0.56
+StringFlav:probQQtoQ 0.078
+StringFlav:ProbStoUD 0.2
+StringFlav:probQQ1toQQ0join 0.0275,0.0275,0.0275,0.0275
+MultiPartonInteractions:pT0Ref 2.15
+BeamRemnants:remnantMode 1
+BeamRemnants:saturation 5
+
+# Correct decay lengths (wrong in PYTHIA8 decay table)
+# Lb
+5122:tau0 = 0.4390
+# Xic0
+4132:tau0 = 0.0455
+# OmegaC
+4332:tau0 = 0.0803
+
+### switch off all decay channels for charm
+411:onMode = off
+421:onMode = off
+431:onMode = off
+4112:onMode = off
+4122:onMode = off
+4232:onMode = off
+4132:onMode = off
+443:onMode = off
+4332:onMode = off
+
+###Semimuonic decays of charm
+
+### D+/- -> mu X
+411:onIfAny = 13
+### D0 -> mu X
+421:onIfAny = 13
+### D_s -> mu X
+431:onIfAny = 13
+### Lambda_c -> mu X
+4122:onIfAny = 13
+### Sigma -> mu X
+4232:onIfAny = 13
+### Sigma_c -> mu X
+4132:onIfAny = 13
+### Omega_c -> mu X
+4332:onIfAny = 13
```

Comment:
Xic baryons are missing here

Reply:
@NicoleBastid I checked https://pdg.lbl.gov/2007/reviews/montecarlorpp.pdf and also the 4232 is a Xic indeed and not a Sigma_c, can you reinclude it please?